### DISCUSSION TITLE
Image formats and minimum sizes

### DISCUSSION THREAD

**Discussion by w2d-ar** [Web-Wise-Dynamics/officialweb#12] [Q&A] [answered]
What will be the allowed image formats and what will be the minimum sizes?

**Comment by thecoderbutcher** [comment, marked as answer]
The ideal formats will be **.webp** but if necessary **.png** will be used, everything will depend on the qualities of the image

As for the size, we will handle a maximum of no more than **100kb** if possible.

**Reply by w2d-ar** [reply]
Perfect!
